crates/ansi-to-html/fuzz/utils/html_interpreter.rs

```diff
@@ -0,0 +1,289 @@
+use html5ever::{
+    local_name, tendril,
+    tokenizer::{
+        BufferQueue, Tag, TagKind, Token, TokenSink, TokenSinkResult, Tokenizer, TokenizerResult,
+    },
+    Attribute, QualName,
+};
+
+use std::{cell::RefCell, collections::BTreeSet, mem, str::FromStr};
+
+/// Convert HTML to runs of stylized text
+pub fn interpret_html(text: &str) -> Vec<StylizedText> {
+    let tokenizer = Tokenizer::new(HtmlInterpreter::default(), Default::default());
+    let mut queue = BufferQueue::default();
+    let text = tendril::Tendril::from_str(text)
+        .unwrap()
+        .try_reinterpret::<tendril::fmt::UTF8>()
+        .unwrap();
+    queue.push_back(text);
+    let res = tokenizer.feed(&mut queue);
```

Comment:
I think `try_reinterpret` is not necessary here. Also, `queue` can be passed as immutable reference.

```suggestion
    let queue = BufferQueue::default();
    let text = tendril::Tendril::from_str(text).unwrap();
    queue.push_back(text);
    let res = tokenizer.feed(&queue);
```